api/sql/schema/380-rule.alterTable.sql

```diff
@@ -58,6 +58,14 @@ BEGIN
     ALTER TABLE [rule].[condition] ADD [name] NVARCHAR(100)
 END
 
+IF NOT EXISTS( SELECT 1 FROM sys.objects WHERE Name = N'ukRuleConditionName' )
+BEGIN
+    DECLARE @updateName NVARCHAR(MAX) = 'UPDATE [rule].[condition] SET [name] = ''Rule '' + CAST(conditionId AS VARCHAR(20)) WHERE [name] IS NULL'
+    EXEC(@updateName)
+    ALTER TABLE [rule].[condition] ALTER COLUMN [name] NVARCHAR(100) NOT NULL
+    ALTER TABLE [rule].[condition] ADD CONSTRAINT [ukRuleConditionName] UNIQUE ([name])
+END
+
```

Comment:
Make the existing `name` column required and unique if the unique constraint is not there yet.